tests/CSharpFunctionalExtensions.FluentAssertions.Tests/MaybeAssertionsTests.cs

```diff
@@ -25,7 +25,7 @@ public void When_Maybe_Is_Expected_To_Have_Value_And_It_Has_Wrong_Value_Should_T
     }
 
     [Fact]
-    public void When_Maybe_Is_Expected_To_Have_Value_And_It_Does_Not_Should_Throw()
+    public void WhenMaybeIsExpectedToHaveValueAndItDoesNotShouldThrow()
```

Comment:
New test naming preference?

Reply:
Just _feels_ better nowadays, overall I could go either way about the underscores.